ci/lepton/model_convergence/configs/recipes/esm2_native_te.yaml

```diff
@@ -3,46 +3,100 @@ defaults:
   - /base
   - _self_
 
-# lepton info
+############################################################
+# lepton job info
+############################################################
 node_group: yo-bom-lepton-001
 num_nodes: 1
 device_type: gpu
 num_devices: 2
 gpu_type: h100-sxm
-total_gpus: ${multiply:${num_devices},${num_nodes}}
 resource_shape: "${device_type}.${num_devices}x${gpu_type}"
+# `total_gpus` calculated from above
+total_gpus: ${multiply:${num_devices},${num_nodes}}
 
+############################################################
 # recipe identifiers
+# mostly used for logging and observability
+############################################################
 recipe_subdir: esm2_native_te
 model_type: esm2
+variant: train # train, finetune
+
+# Core identifiers for filtering
+framework: native # native, accelerate
+parallelism_strategy: fsdp2 # ddp, fsdp2, mfsdp
+precision: fp8 # likely bf16 or fp8
+te_enabled: true
+fp8_enabled: true
 
-# wandb
+# Catchall for additional features/configs
+extras: [] # e.g. [thd]
+
+############################################################
+# wandb info (total_gpus used for group name)
+############################################################
 wandb_init_args:
   project: "test_convergence__recipes__${sanitize:${branch}}"
   group: "${model_type}__${task_cmd}__${total_gpus}__${sanitize:${gpu_type}}"
```

Comment:
maybe  `_${total_gpus}gpus_` since it might be hard to decode from the group name itself